packages/ui/stories/2_components/remoteSelect/remoteSelect.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { RemoteSelect } from '@tidb-community/ui';
+import { api } from '@tidb-community/datasource';
+import { getTitle } from '../utils';
+
+export default {
+  title: getTitle('RemoteSelect'),
+  component: RemoteSelect,
+};
+
+const fetchCompanies = (word) => {
+  return api.org.searchCompany({ word })
+    .then(result => result.data.map(company => ({
+      label: company.name,
+      value: company.name,
+    })));
+};
```

Comment:
If it's not depending on the API call, you may use setTimeout to simulate the asynchronous fetcher method.